examples/error-handling/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "error-handling"
```

Comment:
I still think we should add this to the `customize-extractor-error` crate. Don't see why we need another crate.

We also cannot call it "error handling" because that means something in axum. See https://docs.rs/axum/latest/axum/error_handling/index.html

Reply:
I see now that my comment about not touching existing examples was confusing. I meant not touching other examples than `customize-extractor-error`. Sorry about that.